flocker/node/_model.py

```diff
@@ -189,6 +164,39 @@ class Application(object):
     """
 
 
+@attributes(["dataset", "primary"])
+class Manifestation(object):
+    """
+    A dataset that is mounted on a node.
+
+    :ivar Dataset: The dataset being mounted.
+
+    :ivar bool primary: If true, this is a primary, otherwise it is a replica.
```

Comment:
This appears to be unused.  Lacking any users, it's not clear if this is the right structure for this piece of information.  For example, another approach that might make sense here is to have `Master` and `Replica` types - perhaps with alternate implementations of some `IManifestation` interface.  Or maybe a bool is really the right approach - but it's hard to tell right now.